packages/components/src/TimeInput.test.tsx

```diff
@@ -283,16 +295,3 @@ describe('arrow up and down updates values in segments', () => {
     testArrowValue(6, -3, '12:34:59');
   });
 });
-
-it('updates properly when the value prop is updated', () => {
```

Comment:
Why'd this test case get deleted? Should be able to do this easier with testing library now using `rerender`: https://testing-library.com/docs/example-update-props/